tensorflow/core/common_runtime/direct_session_with_tracking_alloc_test.cc

```diff
@@ -117,13 +117,13 @@ TEST(DirectSessionWithTrackingAllocTest, CostModelTest) {
           // which increments the value of AllocationId.
           // Thus AllocationId becomes more than TF if MKL
           // is used. Now IDs for MKL are 8 more than TF.
-          EXPECT_EQ(21, cm->AllocationId(node, 0));
+          EXPECT_EQ(13, cm->AllocationId(node, 0));
```

Comment:
Why aren't there extra allocations anymore with MKL being used?
Please remove `#if defined(INTEL_MKL) && defined(ENABLE_MKL)` now that the allocation IDs are the same for both MKL and TF. Please also update the comment to explain why they are the same.